src/exampleTypes/index.js

```diff
@@ -37,41 +42,107 @@ export default ({
   ListGroupItem = 'div',
   Icon,
   Select = DefaultSelect,
+  Loading = StripedLoader,
 } = {}) => {
+  let injectNodeAndLoad = (...args) =>
```

Comment:
can we not use spread here since it is only passthrough to `injectTreeNode` and that takes params through a single object.